examples/docker/Dockerfile.Metadata/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0
+
+RUN cd /home && \
+  git clone https://github.com/TileDB-Inc/TileDB-CSharp.git -b bd/ch13349-metadata-example && \
+  dotnet publish TileDB-CSharp/examples/TileDB.Metadata/metadata.csproj -c Release -o /app/example_metadata
```

Comment:
Does this try to upload a package?

Reply:
it will upload nuget packages to the container, build the project and deploy it to the directory /app/example_metadata